README.md

```diff
@@ -81,4 +81,18 @@ git push origin -u add_duck_db
 
 7. [Create a Pull Request (PR)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#creating-the-pull-request) in Github.
 
-8. [Merge your first PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request).
\ No newline at end of file
+8. [Merge your first 
+PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request). 
+
+## Important Metrics and Data Points Available - Workshop 2  
```

Comment:
Great job summarizing our main discussion points from last workshop!

To make your repo less academic and more real-life, I'd suggest rewriting to avoid mentioning the project and fictional client. Instead, we can simply mention a list of features without mentioning the investor use case. Think of it like creating a real open-source library.

I'd suggest this structure:

1. Overview
2. Important Metrics
3. Data source

Lmk what you think!